src/app/context/AccountContext.tsx

```diff
@@ -49,19 +72,32 @@ export function AccountProvider({ children }: { children: React.ReactNode }) {
     getFormattedInCurrency,
   } = useSettings();
 
-  const [account, setAccount] = useState<AccountContextType["account"]>(null);
+  const [account, setAccount] = useState<Partial<AccountInfo> | null>(null);
   const [loading, setLoading] = useState(true);
   const [accountBalance, setAccountBalance] = useState("");
   const [fiatBalance, setFiatBalance] = useState("");
+  const [, setStatus] = useState<StatusRes>();
+
+  const { data: accountInfoRes, mutate } = useSWR<AccountInfoRes>(
```

Comment:
could you rename mutate? like `, mutate: mutateAccountInfo` so it is more obvious what it's doing?